python/test_file_generator.py

```diff
@@ -421,42 +426,42 @@ def v615_3_file(self):
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
 	def v616_file(self):
-		"""Set occupancy to random 4 or blank."""
+		"""Set occupancy to random 4."""
 		name = "v616.txt"
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.occ_type = lar.occ_type.map(lambda x: random.choice(("4", "")))
+		lar.occ_type =  "4"
 		print("writing {name}".format(name=name))
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
 	def v617_file(self):
-		"""Set loan amount to random 0 or blank."""
+		"""Set loan amount to random 0."""
```

Comment:
Can you change the docstrings of the functions that no long have a call to random?